Code/Editor/CryEditPy.cpp

```diff
@@ -210,15 +210,15 @@ namespace
     const char* PyGetCurrentLevelName()
     {
         // Using static member to capture temporary data
-        static AZStd::string tempLevelName;
+        static AZStd::fixed_string<AZ::IO::MaxPathLength> tempLevelName;
```

Comment:
Actually we have a type alias for this already.
It is `AZ::IO::FixedMaxPathString`

Reply:
replaced 23 occurrences